lib/acts_as_taggable_on_mongoid/taggable.rb

```diff
@@ -34,6 +34,18 @@ module Taggable
       #     can be used.  If custom options like `parse` or `parser` are to be used for the default, the value
       #     must be passed in as an array with a hash as the last value.  Like list setters, parsing is assumed.
       #     Example:  default: ["this, is, a, list", parser: ActsAsTaggableOnMongoid::GenericParser]
+      #   * tagger
+      #     Multiple variants
+      #
+      #     true - If simply the value `true`, the tag list supports owners.  No default owner will be used when
+      #            setting or accessing tag_lists
+      #     Hash - A hash of values defining default options for the tag list:
+      #            * default_tagger
+      #              A symbol of a method on the taggable object that owns the HashList which is used to determine
+      #              the default owner for a list if an owner is not specified.
+      #            * tag_list_uses_default_tagger
+      #              true/false indicating if a non-owner method that returns a tag_list should assume the default_tagger
+      #              or return items with no owner.
```

Comment:
Adding these options to the `acts_as_taggable` method will allow us to define tags on the `EndUser` like:

```Ruby
acts_as_taggable_on tagger: { default_tagger: :user, tag_list_uses_default_tagger }
```

Using the tag lists just becomes "normal" looking:
```Ruby
end_user = EndUser.find

end_user.tag_list = "A, list, of, tags"
end_user.save!
end_user.tag_list
# ["A", "list", "of", "tags"]

# The owner of the tags is end_user.user
```